### PR TITLE
Removes Newtonsoft assembly targetting. Fixes flaky integration test.

### PR DESCRIPTION
**Pull request by paterasMSFT**

### CODE REVIEW

File: sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs [hunk 1]
```diff
@@ -467,7 +467,7 @@ public async Task Bug1360227_AddTasksBatchTimeout(bool useJobOperations)
                             batchCli,
                             testName,
                             311,
-                            timeout: TimeSpan.FromSeconds(1),
+                            timeout: TimeSpan.FromSeconds(-1),
```
Comment (dpwatrous) [inline sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs:470]: Seems like we should settle on a timeout somewhere between 1 second and infinite

**Review by dpwatrous — CHANGES_REQUESTED** [review]

File: sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs (hunk 1 above)
Comment (paterasMSFT) [reply sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs:470]: This fixes a flaky test that is supposed to timeout.

Comment (paterasMSFT) [reply sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs:470]: The timespan is eventually added to the start time of the task, to determine an absolute timeout time: https://github.com/Azure/azure-sdk-for-net/blob/4fe2be5d7155f897576d9746692e33dd20360c9b/sdk/batch/Microsoft.Azure.Batch/src/AddTasksWorkflowManager.cs#L103

Comment (dpwatrous) [reply sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs:470]: Thanks, that makes sense - I misinterpreted the -1 as an infinite timeout.

**Review by dpwatrous — APPROVED** [review]

**Pull request merged**